2019/day1/day1_test.go

```diff
@@ -0,0 +1,76 @@
+package day1_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/jessicagreben/adventofcode/2019/day1"
+	"github.com/jessicagreben/adventofcode/2019/pkg/input"
+)
+
+func TestDay1Part1(t *testing.T) {
+	dir, err := os.Getwd()
```

Comment:
Tests are always started in the same folder, so you can directly use `input.ReadFromFile("input.txt")`.